clients/client-java-runtime/build.gradle.kts

```diff
@@ -10,6 +10,14 @@ plugins {
   alias(libs.plugins.shadow)
 }
 
+configurations.all {
+  resolutionStrategy.eachDependency {
+    if (requested.group == "org.apache.logging.log4j") {
+      throw GradleException("Forbidden dependency 'org.apache.logging.log4j' found!")
+    }
+  }
+}
+
```

Comment:
I don't think we have to do this explicit check here, if it is OK by manual check I think it is fine.

Reply:
I prefer to keep this check in case other developers add log4j dependency unexpectedly.